src/objects.c

```diff
@@ -270,6 +272,8 @@ P11PROV_OBJ *p11prov_obj_new(P11PROV_CTX *ctx, CK_SLOT_ID slotid,
     obj->class = class;
     obj->cached = CK_INVALID_HANDLE;
 
+    obj->pub_key_obj = NULL;
```

Comment:
no need for this the zalloc above guarantees everything is NULL by default.

Reply:
Fixed